docs/Writing_easyconfig_files.rst

```diff
@@ -510,12 +510,87 @@ which is equivalent to::
 
   dependencies = [('PnMPI', '1.2.0', '', True)]
 
+
 Using external modules as dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Since EasyBuild v2.1, specifying modules that are not provided via EasyBuild as dependencies is also supported.
 See :ref:`using_external_modules` for more information.
 
+
+Module Extensions
```

Comment:
Just "Extensions" is better, since there's not really extensions to the module file, but extensions to the software provided by the module... :)